### PR TITLE
Update SQLite to 3.14.1 and cleanup wallet.db tables

### PR DESCRIPTION
**Pull request by nbougalis**

### CODE REVIEW

File: src/ripple/app/main/DBInit.cpp [hunk 1]
```diff
@@ -114,216 +114,41 @@ const char* LedgerDBInit[] =
...
-//
 const char* WalletDBInit[] =
 {
-    // Node identity must be persisted for CAS routing and responsibilities.
     "BEGIN TRANSACTION;",
 
+    // A node's identity must be persisted, including
+    // for clustering purposes. This table holds one
+    // entry: the server's unique identity, but the
+    // value can be overriden by specifying a node
```
Comment (wilsonianb) [inline src/ripple/app/main/DBInit.cpp]: `overriden` -> `overridden`

**Comment by wilsonianb** [comment]
LGTM :+1:

**Comment by seelabs** [comment]
:+1: I searched the code to make sure the removed tables we not used.

**Comment by nbougalis** [comment]
Merged as 4df24c0e8e336dbd5660b5748e740449a6753d86 and a9b3042d7e2784a26dda5959ac09140ba8f39f6c

**Pull request closed without merging**